go.mod

```diff
@@ -5,17 +5,15 @@ go 1.14
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/aerospike/aerospike-client-go v2.10.0+incompatible
-	github.com/beorn7/perks v1.0.0 // indirect
 	github.com/gobwas/glob v0.2.3
-	github.com/golang/protobuf v1.3.2-0.20190517061210-b285ee9cfc6c // indirect
+	github.com/golang/protobuf v1.4.2 // indirect
 	github.com/jameskeane/bcrypt v0.0.0-20120420032655-c3cd44c1e20f
 	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
-	github.com/prometheus/client_golang v0.9.4-0.20190605154215-ea8c935f204c
```

Comment:
I'm a bit worried about these deleted lines. How were they removed? By `go mod` itself?

Reply:
Yes, by `go mod` itself. The removed ones are unused dependencies (some of them are upgraded).